### DISCUSSION TITLE
Microgrid

### DISCUSSION THREAD

**Discussion by raygituser** [fredrikofstad/RL-Microgrid-Coordination#2] [General]
Since we have binary actions and the microgrid class doesnt have any way deviding the total energy into the energy you sell ,use for the load and save to the battery then if for example wind energy action load is 1 and sell back for wind is 1 all of the energy goes to the energyload or to sell,and sinse the energy load is so big, energy perduced only contributes to the enery load.?

The way I did this was after calculatating total energy generated I set 70 % of it to energyload and so on.

And I also think unit operational cost are to small?

Did you guys try to plot things?, I am not getting nice plots.

**Comment by raygituser** [comment]
All the constants insentivises the microgrid to only buy from the grid.I exprementing with the constants you have to have a lot of windmills,you a have increase the solarPV area or increase the number of generators to actually see some improvment on the cost.Is this what you guys see as well?

**Comment by hui-zhang-adabella** [comment]
Hei guys, we have Zhengyu joining in our group, hurra!!!

**Comment by fredrikofstad** [comment]
I'll do some tests and plot my results some time today!
